packages/components/src/components/trace/TraceFilter.vue

```diff
@@ -0,0 +1,354 @@
+<template>
+  <div class="trace-filter">
+    <form ref="form" :class="formClasses" @submit.prevent="onFormSubmit">
+      <span class="trace-filter__prefix">
+        <SearchIcon />
+      </span>
+      <input
+        ref="input"
+        v-model="filterValue"
+        class="trace-filter__input"
+        type="text"
+        autocomplete="off"
+        placeholder="search events..."
+        @focus="onInputFocus"
+      />
+      <span v-if="filterValue" class="trace-filter__suffix" @click="clearValue">
+        <CloseThinIcon />
+      </span>
+      <ul
+        ref="suggestionsList"
+        v-if="suggestions.length"
+        class="trace-filter__suggestions"
+      >
+        <li
+          :class="getSuggestionClasses(index)"
+          v-for="(item, index) in suggestions"
```

Comment:
`suggestions` should update as the user inputs text into the search bar. The selected list item should be the best match, similar to the root object filters.

https://user-images.githubusercontent.com/8737782/152194221-c856ac03-88d0-439a-8abd-f1e225d4fe59.mp4

Reply:
Done https://github.com/applandinc/appmap-js/pull/534/commits/ab513bf07e98b30335eb29f80bcc490490ab1d94